docs/team/icytornado.md

```diff
@@ -5,16 +5,55 @@ title: Zhang Peng's Project Portfolio Page
 
 Given below are my contributions to the project.
 
-* **Code contributed**:
+* **Code contributed**: [RepoSense] (https://nus-cs2103-ay2021s2.github.io/tp-dashboard/#breakdown=true&search=icytornado)
+
+* **Features Implemented**:
+  * Implementation of the `edit-appt` feature:
+    - *Description*:
+        - `edit-appt` command allow the user to edit an existing appointment by changing the fields for the appointment.
+        - `edit-appt` + `index` + `field` to edit => edit the specified field of the appointment specified by the appointment index.
+        - the `field` to edit is preceded by its related prefix, i.e. `pt/` for patient, `dr/` for doctor, `at/` for appointment start date and time, `to/` for appointment end date and time, `dur/` for appointment duration, `/t` for appointment tags.
+    
+  * Implementation of the `Doctor` feature
+    - *Description*:
+        - Created the `Doctor` class and implemented its related command features including
+          `delete-doctor`, `edit-doctor`, `add-doctor`, `clear-doctor`, `list-doctor`.
+            
+  * Added some relevant test classes for `Doctor`.
 
-* **Project management**:
 
 * **Documentation**:
+  * User Guide:
+    - Added documentation on `edit-appt` and `edit-patient` feature.
+        - Pull request [#50](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/50)
+  * Developer Guide:
+    - Updated the UML diagrams and description for the Model component under the Design section:
+      * [Model Class Diagram](https://github.com/AY2021S2-CS2103-W17-2/tp/blob/master/docs/diagrams/ModelClassDiagram.puml)
+      * [Better Person Model Class Diagram](https://github.com/AY2021S2-CS2103-W17-2/tp/blob/master/docs/diagrams/BetterPersonModelClassDiagram.puml)
+      * [Better Appointment Model Class Diagram](https://github.com/AY2021S2-CS2103-W17-2/tp/blob/master/docs/diagrams/BetterAppointmentModelClassDiagram.puml)
+        - Pull request [#163](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/163)
+    - Added user stories for some command features.
+       - Pull request [#154](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/154)
+    - Added Use Case documentation on `edit-patient`, `find-patient`, `delete-doctor`, `edit-doctor`, `find-doctor`, `edit-appt` features.
+       - Pull request [#162](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/162)
+    - Added Instructions for Manual Testing for `edit-patient`, `find-patient`, `delete-doctor`, `edit-doctor`, `find-doctor`, `edit-appt` features.
+       - Pull request [#169](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/169)
+
+
+* **Project management**:
+   * Updated a few PRs with the relevant tags and milestones for the project
```

Comment:
these PRs that you updated were opened by you and were supposed to be tagged by you anyway, and not part of project management

Reply:
will update accordingly